components/hab/src/command/plan/init.rs

```diff
@@ -143,15 +163,20 @@ pub fn start(
     };
     ui.para(
         "The `hooks` directory is where you can create a number of automation hooks into \
-               your habitat. There are several hooks to create and tweak! See the full list \
-               with info here: https://www.habitat.sh/docs/reference/plan-syntax/#hooks",
+        your habitat.",
+    )?;
+
+    ui.para(
+        "For more information on any of the generated files: \
+        https://www.habitat.sh/docs/reference/plan-syntax/#basic-settings \
+        https://www.habitat.sh/docs/reference/plan-syntax/#runtime-configuration-settings \
+        https://www.habitat.sh/docs/reference/plan-syntax/#hooks \
+        https://www.habitat.sh/docs/reference/plan-syntax/#callbacks",
```

Comment:
I much prefer the output containing URL links to being embedding in the `plan.sh`. Nicely done!